tmt/steps/execute/internal.py

```diff
@@ -97,17 +102,20 @@ def options(cls, how=None):
         options.append(click.option(
             '--no-progress-bar', is_flag=True,
             help='Disable interactive progress bar showing the current test.'))
-        return options + super().options(how)
+        return cast(List[tmt.options.ClickOptionDecoratorType], options) + \
```

Comment:
No need to use `cast()` here, see how this method is implemented in e.g. [`polarion.py`](https://github.com/teemtee/tmt/blob/main/tmt/steps/report/polarion.py#L24).

Reply:
@guy9050 this is the same comment I left you in https://github.com/teemtee/tmt/pull/1302

Reply:
Changed. Thanks.